.gitignore

```diff
@@ -11,3 +11,4 @@ tags
 .idea/
 .vscode/
 compile_commands.json
+.cache
```

Comment:
Please, submit this single patch separately. It has nothing to do with the PR subject